api/v1alpha1/toolchainconfig_types.go

```diff
@@ -194,6 +194,10 @@ type NotificationsConfig struct {
 	// +optional
 	AdminEmail *string `json:"adminEmail,omitempty"`
 
+	// NotificationEnvironment is to distinguish the cluster environment being run in, eg: whether it is for sandbox or stonesoup
+	// +optional
+	NotificationEnvironment *string `json:"notificationEnvironment,omitempty"`
```

Comment:
What if we rename it something more specific to templates? The environment terms is quite overloaded.
```suggestion
	// NotificationTemplateSetName defines the set of notification templates. Different Sandbox instances can use different notification templates. For example Dev Sandbox and AppStudio instances use different templates. By default the "sandbox" template set name is used.
	// +optional
	NotificationTemplateSetName *string `json:"notificationTemplateSetName,omitempty"`
```